tests/test_staticfiles.py

```diff
@@ -68,13 +68,13 @@ def test_staticfiles_with_package(test_client_factory):
     client = test_client_factory(app)
     response = client.get("/example.txt")
     assert response.status_code == 200
-    assert response.text == "123\n"
+    assert response.text.replace("\r\n", "\n") == "123\n"
```

Comment:
What's this?

Reply:
Helps test run on windows, where example.txt has this encoding.  Not-essential to this PR.

Reply:
I guess it depends on how it is checked out.  I cloned it in windows, then was testing it in WSL.  The file then has \r\n endings.  I guess it would be best to list this file in .gitattributes as having  \n line endings.   Anyway, I've removed this change from the PR